hmac/Cargo.toml

```diff
@@ -17,10 +17,10 @@ digest = { version = "0.10.3", features = ["mac"] }
 [dev-dependencies]
 digest = { version = "0.10", features = ["dev"] }
 md-5 = { version = "0.10", default-features = false }
-sha-1 = { version = "0.10", default-features = false }
+sha1 = { version = "0.10", default-features = false }
 sha2 = { version = "0.10", default-features = false }
 streebog = { version = "0.10", default-features = false }
-hex-literal = "0.2.2"
+hex-literal = "0.3.4"
```

Comment:
`hex-literal v0.3` has higher MSRV than that of the crate, so you should revert this change.